allure-adapter-plugin/src/main/kotlin/io/qameta/allure/gradle/adapter/AllureAdapterExtension.kt

```diff
@@ -121,7 +121,9 @@ open class AllureAdapterExtension @Inject constructor(
         task.run {
             // Each task should store results in its own folder
             // End user should not depend on the folder name, so we do not expose it
-            val rawResults = project.layout.buildDirectory.dir("allure-results").get().asFile
+            // Note: it is very important that results directory is named `allure-results`, so the folder is automatically
+            // detected by Allure Jenkins plugin.
+            val rawResults = project.layout.buildDirectory.dir("allure-raw-results/${task.name}/allure-results").get().asFile
```

Comment:
An alternative option is `allure-raw-results/${task.name}/build/allure-results` so the ones who hard-code `build/allure-results` would find the results folder as well.

Reply:
(╯°□°)╯︵ ┻━┻
https://github.com/search?q=%22build%2Fallure-results%22&type=code